package/doc/sphinx/source/documentation_pages/topology_modules.rst

```diff
@@ -19,6 +19,8 @@ properly recognize a file format then a user can explicitly set the
 topology file format in the *topology_format* keyword argument to
 :class:`~MDAnalysis.core.universe.Universe`.
 
+Note: ``masses`` and ``types`` guessing have been removed from all parsers and now takes place inside the :class:`~MDAnalysis.core.universe.Universe`.
```

Comment:
Hmm, I'm not sure this is the best place for such a note -- it would quickly become outdated with newer versions and we're unlikely to remember to update it. Perhaps we can remove this and just stay with the `versionchanged` notes in the classes?